package.json

```diff
@@ -11,20 +11,38 @@
   ],
   "packageManager": "yarn@4.5.1",
   "devDependencies": {
+    "@babel/preset-typescript": "7.25.9",
+    "@semantic-release/changelog": "^6.0.3",
+    "@semantic-release/exec": "^6.0.3",
+    "@semantic-release/git": "^10.0.1",
+    "@semantic-release/github": "^11.0.1",
     "html-webpack-plugin": "^5.6.3",
+    "jest": "29.7.0",
     "node-gyp": "^10.2.0",
+    "semantic-release": "^24.2.0",
     "typescript": "^5.6.3",
     "webpack": "^5.96.1",
     "webpack-cli": "^5.1.4",
     "webpack-dev-server": "^5.1.0"
   },
   "scripts": {
+    "test:unit": "yarn run test:tooling && yarn run test:cc-widgets",
+    "test:tooling": "jest --coverage",
+    "test:cc-widgets": "yarn run test:store && yarn run test:widgets",
+    "test:store": "yarn workspace @webex/cc-store test:unit",
+    "test:widgets": "yarn workspace @webex/cc-station-login test:unit && yarn workspace @webex/cc-user-state test:unit",
```

Comment:
This is a suggestion for improvement. We could use `yarn workspaces foreach` when we'd like to run the same command across different packages